firebase/auth/v1/auth_event_data.go

```diff
@@ -24,7 +24,7 @@ type AuthEventData struct {
 	EmailVerified *bool           `json:"emailVerified,omitempty"` // Whether or not the user's primary email is verified.
 	Metadata      *Metadata       `json:"metadata,omitempty"`      // Additional metadata about the user.
 	PhoneNumber   *string         `json:"phoneNumber,omitempty"`   // The user's phone number.
-	PhotoURL      *string         `json:"photoUrl,omitempty"`      // The user's photo URL.
+	PhotoURL      *string         `json:"photoURL,omitempty"`      // The user's photo URL.
```

Comment:
I'm surprised the JSON uses capital "URL" in these property names, but sgtm.

Reply:
https://github.com/googleapis/google-cloudevents/blob/3fe5d3010c39a695436eb83c560dc703be6cac36/proto/google/events/firebase/auth/v1/data.proto#L39

Yeah, it's funky. The proto docs aren't clear exactly the spelling in this case.

I checked the Firebase Auth docs, and the JSON field is indeed `photoURL` [0], so I'm going to assume that this is correct (in the proto2jsonschema tool/underlying library) for now. These events haven't ever been GA (in GCF), so we also have more flexibility in the future for changing these events in the future.

[0] https://firebase.google.com/docs/auth/web/manage-users#get_a_users_profile